project/query.py

```diff
@@ -0,0 +1,33 @@
+"""Query class."""
+
+import os
+import tweepy
+
+class Query(object): # pylint: disable=too-few-public-methods
+    """Class to query multiple API's."""
+    def __init__(self):
+        # Authenticates multiple API's
+
+        # Twitter authentification process
+        twit_consumer_key = os.environ.get('TWITTER_CONSUMER_KEY')
```

Comment:
Prefer loading these values in Config during initialization. Then can access with current_app:

from flask import current_app as app
...
app.config['TWITTER_CONSUMER_KEY']